cli/wait.go

```diff
@@ -0,0 +1,72 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/spf13/cobra"
+)
+
+// waitDescription is used to describe wait command in detail and auto generate command doc.
+var waitDescription = "Block until one or more containers stop, then print their exit codes. " +
+	"If container state is already stopped, the command will return exit code immediately. " +
+	"On a successful stop, the exit code of the container is returned. "
+
+// WaitCommand is used to implement 'wait' command.
+type WaitCommand struct {
+	baseCommand
+}
+
+// Init initializes wait command.
+func (wait *WaitCommand) Init(c *Cli) {
+	wait.cli = c
+	wait.cmd = &cobra.Command{
+		Use:   "wait CONTAINER [CONTAINER...]",
+		Short: "Block until one or more containers stop, then print their exit codes",
+		Long:  waitDescription,
+		Args:  cobra.MinimumNArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return wait.runWait(args)
+		},
+		Example: waitExamples(),
+	}
+}
+
+// runWait is the entry of wait command.
+func (wait *WaitCommand) runWait(args []string) error {
+	ctx := context.Background()
+	apiClient := wait.cli.Client()
+
+	var errs []string
+	for _, name := range args {
+		response, err := apiClient.ContainerWait(ctx, name)
+		if err != nil {
+			errs = append(errs, "container exitcode: "+strconv.FormatInt(response.StatusCode, 10)+
```

Comment:
I think we can just show the error without response, because in golang we don't consume the data if err != nil.